.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "lib2813"]
 	path = lib2813
-	url = https://github.com/Prospect-Robotics/lib2813.git
+	url = https://github.com/Prospect-Robotics/lib2813
```

Comment:
not sure if we need/want this change

Reply:
Looks like this may be accidental; I see the reverse change in 7d6d95d.

I'll push a commit to fix this.